lib/core/presentation/pages/chat/chat_message/widgets/chat_input/reply_content_widget.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+
+class ReplyContent extends StatelessWidget {
+  final Event replyEvent;
+  final bool ownMessage;
+  final Timeline? timeline;
+
+  const ReplyContent(
+    this.replyEvent, {
+    this.ownMessage = false,
+    Key? key,
+    this.timeline,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    Widget replyBody;
+    final timeline = this.timeline;
+    final displayEvent =
+        timeline != null ? replyEvent.getDisplayEvent(timeline) : replyEvent;
+    final fontSize = Typo.medium.fontSize!;
+
+    replyBody = Text(
+      displayEvent.calcLocalizedBodyFallback(
+        MatrixDefaultLocalizations(),
+        withSenderNamePrefix: false,
+        hideReply: true,
+      ),
+      overflow: TextOverflow.ellipsis,
+      maxLines: 1,
+    );
+
+    return Row(
+      mainAxisSize: MainAxisSize.min,
+      children: <Widget>[
+        Container(
+          width: 3,
+          height: fontSize * 2 + 6,
```

Comment:
Why 6, this 6 using in SizeBox, and width = 6 / 3 = 3 
=> It's better define some variables for this number to know what it's doing